### PR TITLE
Add component & foundation illustrations

### PR DESCRIPTION
**Pull request by MelSumner**
### :pushpin: Summary

* Implements title-inference naming of illustration images
* Disables fingerprinting (at least for now) for illustrations since this approach doesn't work with fingerprinting
* Adds all the images for components & foundations

**Note** There are in flight PRs that will resolve form illustrations that don't work correctly (they are being renamed)

https://hds-website-git-melsumner-update-component-art-hashicorp.vercel.app/components
https://hds-website-git-melsumner-update-component-art-hashicorp.vercel.app/foundations

Screen Shot 2022-12-23 at 09 29 22
Screen Shot 2022-12-23 at 09 29 24

### CODE REVIEW

File: website/app/components/doc/cards/card.hbs [hunk 1]
```diff
@@ -1,7 +1,14 @@
 <li class={{this.classNames}} ...attributes>
   {{#if @route}}
     <LinkTo class="doc-cards-card__link" @route={{@route}} @model={{@model}}>
-      <img class="doc-cards-card__image" src={{@image}} alt="" role="presentation" />
+      <img
+        class="doc-cards-card__image"
+        src={{@image}}
+        alt={{(concat @title " - " @caption)}}
```
Comment (Dhaulagiri) [inline website/app/components/doc/cards/card.hbs:7]: @MelSumner this triggered a linting error so it seems like I should undo this. Can you confirm?

File: website/ember-cli-build.js [hunk 2]
```diff
@@ -21,6 +21,8 @@ module.exports = function (defaults) {
     fingerprint: {
       // override defaults to also include json files which our markdown is compiled to. without this images don't render properly.
       replaceExtensions: ['html', 'css', 'js', 'json'],
+      exclude: ['illustrations/foundation', 'illustrations/component'],
+      enabled: true,
```
Comment (Dhaulagiri) [inline website/ember-cli-build.js:25]:
```suggestion

```

**Review by jorytindall — APPROVED** [review]
🚀

**Review by KristinLBradley — APPROVED** [review]

File: website/app/components/doc/cards/card.hbs (hunk 1 above)
Comment (MelSumner) [reply website/app/components/doc/cards/card.hbs]: @Dhaulagiri what was the linting error? Might be a false positive, so I'd just add an inline ignore if so. But the way that we use most of the card images, they are superfluous anyway so I'd probably do an empty alt and a role of none.

Comment (Dhaulagiri) [reply website/app/components/doc/cards/card.hbs]: i recall that we discussed this at the time and i removed the alt tag entirely

**Pull request merged**